src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,204 @@
+package duke.command;
+
+import duke.storage.DukeIOException;
+import duke.storage.Storage;
+import duke.task.*;
+
+import java.util.List;
+
+/**
+ * The class Command denotes a command object.
+ *
+ * @author Alvin Chee
+ */
+public class Command {
+
+    private CommandType commandType;
+    private String taskInfo;
+    private boolean isExit;
+
+    /**
+     * Constructs a command
+     *
+     * @param commandType  Type of command.
+     * @param taskInfo  Description of task.
+     */
+    public Command(CommandType commandType, String taskInfo) {
+        this.commandType = commandType;
+        this.taskInfo = taskInfo;
+    }
+
+    /**
+     * Executes a command
+     *
+     * @param tasks  The list of tasks for user.
+     * @param storage   Storage object to store task list.
+     */
+    public void execute(TaskList tasks, Storage storage) {
+        try {
+            switch (commandType) {
+            case BYE:
+                isExit = true;
+                break;
+            case PRINT:
+                printList(tasks);
+                break;
+            case MARKTASKDONE:

```

Comment:
You might want to use the low dash to separate words in this enum value.

Reply:
Thank you for your comment. May I know what do you mean by low dash?